src/terminal_chat_console.cpp

```diff
@@ -347,10 +347,7 @@ void TerminalChatConsole::step(int ch)
 			continue;
 
 		std::wstring error_message = utf8_to_wide(Logger::getLevelLabel(p.first));
-		if (!g_settings->getBool("disable_escape_sequences")) {
-			error_message = std::wstring(L"\x1b(c@red)").append(error_message)
-				.append(L"\x1b(c@white)");
-		}
```

Comment:
Is this change intended?

Reply:
no sorry that was meant for/should be in another PR.